src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const message = document.createElement('div');
+const body = document.body;
+
+new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+}).then(() => {
+  message.classList.add('message');
+  message.innerHTML = `Promise was resolved!`;
+  body.appendChild(message);
+});
+
+new Promise((resolve, reject) => {
```

Comment:
here – as well (I believe linter would not complain)
```suggestion
new Promise((_, reject) => {
```